HeliosCLI/Makefile

```diff
@@ -87,12 +101,18 @@ all: $(TARGETS)
 # unit test target
 tests: $(TESTS)
 
-# target for vortex lib
+# target for helios cli
 helios: compute_version $(DEPS)
-	$(CC) $(CFLAGS) $(DEPS) -o $@ $(LLIBS)
+	@mkdir -p $(OUTPUT_DIR)
+	$(CC) $(CFLAGS) $(OBJS) -o $(FINAL_TARGET) $(LLIBS)
 
-# catch-all make target to generate .o and .d files
-%.o: %.cpp
+# Rules for building object files
+$(BUILD_DIR)/Helios/%.o: ../Helios/%.cpp

```

Comment:
so in theory this shouldn't exist, all building of HeliosLib stuff should be in the makefile that is in that folder